app/controllers/shared/api.php

```diff
@@ -57,8 +57,72 @@
     return $label;
 };
 
-$databaseListener = function (string $event, Document $document, Document $project, Usage $queueForUsage, Database $dbForProject) {
+$eventDatabaseListener = function (string $event, Document $document, Event $queueForEvents, Response $response, Func $queueForFunctions, Document $project) {
+    // Set event empty to avoid triggering events by default
+    $queueForEvents->setEvent('');
 
+    if ($document->getCollection() === 'users' && $event === Database::EVENT_DOCUMENT_CREATE) {
```

Comment:
lets use a switch case here, and we can also remove the second condition right? Since this listener is only called on document creates?